.github/workflows/test-win-installer.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test Win Installer
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [Build Inviwo]
+    types: 
+      - completed
+
+jobs:
+  test-win-installer:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: windows-latest
+    
+    steps:
+    - name: Download artifact
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        name: inviwo-installer-win
+        workflow: inviwo.yml
+
+    - name: Run Installer
+      shell: pwsh
+      run: |
+        $exe = (Get-ChildItem -Name Inviwo-*.exe)
+        echo $exe
+        Invoke-expression "./$exe /S"
+        Start-Sleep -Seconds 180
```

Comment:
Would be good to comment on this 180 seconds. Trial and error value? What does it imply, that it will fail if the installer does not complete within 180 seconds?